src/connectors/impls/kafka/consumer.rs

```diff
@@ -744,14 +742,14 @@ impl Source for KafkaConsumerSource {
         Ok(())
     }
 
-    async fn on_cb_close(&mut self, _ctx: &SourceContext) -> Result<()> {
+    async fn on_cb_trigger(&mut self, _ctx: &SourceContext) -> Result<()> {
```

Comment:
talked about it with mattias, problem was that the original naming and the events were different so the event `CB::Trigger` called `on_cb_close` which makes no sense, we ended up standardizing on `Trigger` instead of `Close` so it's not disconnected